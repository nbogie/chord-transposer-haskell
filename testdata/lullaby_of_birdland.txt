Lullaby of Birdland
Words & Music by George David Weiss & George Shearing
Recorded by Mel Torme, 1963

Em           C#m7-5    F#7-9       B7
Lullaby of birdland  - that's what I

Em7    CM7   Am7      D9
Always hear when you sigh;

Bm7          Em7
Never in my word land

Am7            Cdim      G   C9        F#m7-5       B7
Could there be ways to reveal    in a phrase how I feel.

Em            C#m7-5    F#m7-9   B7
Have you ever heard two turtle doves

Em7      CM7  AM7       D9
Bill and coo when they love?

Bm7                Em7
That's the kind of magic

Am7      Cdim           G   D7           G     G/F#
Music we make with our lips     when we kiss.

Bridge:
E9            E7-9       Am      Am7
And there's a weepy old willow -

D9        Cdim          G     G/F#
He really knows how to cry!

E9             E7-9      Am      Am7
That's how I'd cry on my pillow

D9            Cdim        G              B7
If you should tell me farewell and good-bye.

Em        C#m7-5      F#7-9    B7
Lullaby of birdland - whisper low,

Em7     CM7    Am7       D9
Kiss me sweet, and we'll go

Bm7            Em7
Flyin' high in birdland,

Am7         Cdim    G    C9      F#m7-5           B7
High in the sky up above    all because we're in love.

Repeat Bridge. Repeat last verse. 

Coda:
Am7        Cdim      G
High in the sky up above

 Am7    D9  Cdim   G     C9    Am7   G#7   G
All because we're- in - love.
