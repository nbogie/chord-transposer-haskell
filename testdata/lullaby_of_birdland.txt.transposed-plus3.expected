Lullaby of Birdland
Words & Music by George David Weiss & George Shearing
Recorded by Mel Torme, 1963

Gm           Em7-5     A7-9        D7
Lullaby of birdland  - that's what I

Gm7    D#M7  Cm7      F9
Always hear when you sigh;

Dm7          Gm7
Never in my word land

Cm7            D#dim     A#  D#9       Am7-5        D7
Could there be ways to reveal    in a phrase how I feel.

Gm            Em7-5     Am7-9    D7
Have you ever heard two turtle doves

Gm7      D#M7 CM7       F9
Bill and coo when they love?

Dm7                Gm7
That's the kind of magic

Cm7      D#dim          A#  F7           A#    A#/A
Music we make with our lips     when we kiss.

Bridge:
G9            G7-9       Cm      Cm7
And there's a weepy old willow -

F9        D#dim         A#    A#/A
He really knows how to cry!

G9             G7-9      Cm      Cm7
That's how I'd cry on my pillow

F9            D#dim       A#             D7
If you should tell me farewell and good-bye.

Gm        Em7-5       A7-9     D7
Lullaby of birdland - whisper low,

Gm7     D#M7   Cm7       F9
Kiss me sweet, and we'll go

Dm7            Gm7
Flyin' high in birdland,

Cm7         D#dim   A#   D#9     Am7-5            D7
High in the sky up above    all because we're in love.

Repeat Bridge. Repeat last verse. 

Coda:
Cm7        D#dim     A#
High in the sky up above

Cm7     F9  D#dim  A#    D#9   Cm7   B7    A#
All because we're- in - love.
